Roslyn.sln

```diff
@@ -1880,8 +1880,12 @@ Global
 		{2801F82B-78CE-4BAE-B06F-537574751E2E}.Release|x86.Build.0 = Release|x86
 		{9B25E472-DF94-4E24-9F5D-E487CE5A91FB}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{9B25E472-DF94-4E24-9F5D-E487CE5A91FB}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{9B25E472-DF94-4E24-9F5D-E487CE5A91FB}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{9B25E472-DF94-4E24-9F5D-E487CE5A91FB}.Debug|x86.Build.0 = Debug|Any CPU
```

Comment:
Any idea what happened here?

Reply:
this is happening in all my branches.  no clue why.